website/content/en/docs/concepts/_index.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Concepts"
+linkTitle: "Concepts"
+weight: 35
+---
+
+By adding Karpenter to a Kubernetes cluster you can dramatically improve the efficiency and cost of running workloads on that cluster.
+Instead of introducing lots of new concepts and configuration, Karpenter is tightly integrated with Kubernetes features to make sure that the right types and amounts of compute resources are available to pods as they are needed.
+Karpenter carries out its duties by:
+
+* Watching for pods that the Kubernetes scheduler has marked as unschedulable
+* Evaluating the requirements (capacity, features, and taints) identified by the unscheduable pods
+* Provisioning nodes that meet the requirements of the unschedulable pods
+* Scheduling the unschedulable pods to run on the new nodes
+* Removing the nodes when the nodes are no longer needed
+
+In many cases, you can configure a simple Karpenter provisioner when it is first installed and not change it again. 
+Other times, you might want to continue to tweak the provisioner or even create multiple provisioners for a cluster where you tailor the requirements of different teams to use specific instance types, zones, or computer architectures.
+Likewise, you might want to instruct the provisioner to take advantage of underlying cloud features, such as AWS spot instances.
+
+As someone deploying pod workloads, you can make specific requests for capacity and features you want from the nodes running your pods.
+If the Kubernetes scheduler is unable to meet those requirements, Karpenter is designed to quickly create the best possible nodes to meet those needs.
+
+## Karpenter concepts
+
+The following are some key concepts you should understand about Karpenter.
+
+### Karpenter provisioners
+
+A Karpenter provisioner controller runs on a Kubernetes cluster.
+Its job is to add nodes to handle unschedulable pods, schedule pods on those nodes, and remove the nodes when they are not needed.
+The controller's work is configured through [Provisioner CRDs](/docs/provisioner-crd/).
+
+Rather than take over the process of provisioning nodes and scheduling pods, the provisioner controller integrates with the Kubernetes scheduler, cluster autoscaler, and related Kubernetes features to make sure that pods are properly and efficiently deployed.
```

Comment:
There are a couple inaccuracies here.
>Rather than take over the process of provisioning nodes and scheduling pods, 

I'd remove this rather statement as it may not always be true for this project ;)

>the provisioner controller integrates with the Kubernetes scheduler, cluster autoscaler,

We don't integrate with the cluster autoscaler. They don't play well together unless carefully configured.

>and related Kubernetes features to make sure that pods are properly and efficiently deployed.
 
This reads as a non sequitur to me